vcd/resource_vcd_vapp.go

```diff
@@ -249,6 +255,26 @@ func resourceVcdVAppCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if _, ok := d.GetOk("properties"); ok {
+		// TODO - this is just for testing. Real issue must be solved
```

Comment:
I have put this in during development and spotted only now. I need to fix this yet.

Reply:
This is no longer used and 
`err = vapp.BlockWhileStatus("UNRESOLVED", vcdClient.MaxRetryTimeout)` was introduced